personalities/sctfe/storage.go

```diff
@@ -104,9 +104,7 @@ type cachedIssuerStorage struct {
 // Exists checks if the key exists in the local cache, if not checks in the underlying storage.
 // If it finds it there, caches the key locally.
 func (c *cachedIssuerStorage) Exists(ctx context.Context, key []byte) (bool, error) {
-	c.RLock()
```

Comment:
Aren't you back in `panic` territory now, though?
You'll need to `c.Lock()` here still, I think?

Reply:
Huh - I thought only two concurrent writes would fail.

Reply:
No, unprotected `map` are not safe for anything other than concurrent reads; a concurrent read & write would be bad too.